### ISSUE TITLE
Job hangs indefinitely if output for thumbnailmap encode already exists

### ISSUE THREAD

**Issue by grusell** [svt/encore#23]
**Describe the bug**
If one for whatever reason run multiple jobs with identical outputFolder/baseName, and the profile includes a thumbnailmap encode, the job never finishes.

This seems to be caused by the postprocessing step in Thumbnailmapencode. If the output file for the thumbnailmap postprocessing already exists, ffmpeg process will hang forever waiting for user confirmation that the file should be overwritten.

**To Reproduce**
1. Start a job using a profile with a thumbnailmap encode
2. When the job finishes, start another identical job
3. Job never finishes

**Expected behavior**
Job should finish and overwrite the old thumbnailmap.

**Suggested fix**

```diff --git a/encore-common/src/main/kotlin/se/svt/oss/encore/model/profile/ThumbnailMapEncode.kt b/encore-common/src/main/kotlin/se/svt/oss/encore/model/profile/ThumbnailMapEncode.kt
index d3cd406..2ce7e73 100644
--- a/encore-common/src/main/kotlin/se/svt/oss/encore/model/profile/ThumbnailMapEncode.kt
+++ b/encore-common/src/main/kotlin/se/svt/oss/encore/model/profile/ThumbnailMapEncode.kt
@@ -80,6 +80,7 @@ data class ThumbnailMapEncode(
                     val targetFile = outputFolder.resolve("${job.baseName}$suffix.$format")
                     val process = ProcessBuilder(
                         "ffmpeg",
+                        "-y",
                         "-i",
                         "${job.baseName}$suffix%04d.$format",
                         "-vf",```

**Comment by fhermansson** [comment]
Thank you for reporting! We will look at it for the next release.

**Comment by fhermansson** [comment]
It's fixed in v0.2.2
